packages/xo-web/src/common/xo/create-snapshot-vms-modal/index.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'intl'
+import React from 'react'
+import BaseComponent from 'base-component'
+import SingleLineRow from 'single-line-row'
+import { Col } from 'grid'
+import { isEmpty, map } from 'lodash'
+import { createGetObjectsOfType } from 'selectors'
+import { buildTemplate, connectStore } from 'utils'
+
+@connectStore(
+  () => ({
+    vms: createGetObjectsOfType('VM').pick((_, props) => props.vms),
+  }),
+  { withRef: true }
+)
+export default class SnapshotVmsModalBody extends BaseComponent {
+  state = { namePattern: `{name}_{date}` }
+
+  get value() {
+    const { namePattern } = this.state
+    const _buildTemplate =
+      !isEmpty(namePattern) &&
```

Comment:
`isEmpty`?

Reply:
it can be empty if the `input` is empty.

Reply:
But why do you use `isEmpty` on a string?

Reply:
it's a mistake :/